packages/components/src/components/page-error/page-error.jsx

```diff
@@ -69,11 +69,34 @@ const PageError = ({
                         )}
                     </span>
                 </div>
-                {redirect_label && (
-                    <ButtonLink className='dc-page-error__btn' to={redirect_url} onClick={onClickHandler} size='large'>
-                        <span className='dc-page-error__btn-text dc-btn__text'>{redirect_label}</span>
-                    </ButtonLink>
-                )}
+                <div className='dc-page-error__btn-wrapper'>
+                    {Array.isArray(redirect_url) ? (
+                        redirect_url.map((url, index) => {
+                            return (
+                                <ButtonLink
+                                    className='dc-page-error__btn'
+                                    to={url}
+                                    onClick={onClickHandler}
+                                    size='large'
+                                    key={index}
+                                >
+                                    <span className='dc-page-error__btn-text dc-btn__text'>
+                                        {redirect_label[index] || redirect_label}
+                                    </span>
```

Comment:
`Text` component?

Reply:
I'll check if no one worked on this file yet, I can change it. Otherwise this might create conflicts.